docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -25,19 +25,22 @@ public class Client {
         HazelcastInstance instance = getHazelcastInstance();
         UserCodeNamespacesConfig parentConfig = instance.getConfig().getNamespacesConfig(); <1>
         UserCodeNamespaceConfig namespaceConfig = new UserCodeNamespaceConfig("my_namespace"); <2>
-        namespaceConfig.addClass(MyRequireClass.class); <3>
-        parentConfig.addNamespaceConfig(namespaceConfig); <4>
+            namespaceConfig.addClass(MyRequireClass.class); <3>
+            UserCodeNamespaceConfig ucn = new UserCodeNamespaceConfig("my_namespace")
+                .addClass("url/to/the/class/file", "classId"); <4>
```

Comment:
For the Jar that you add below on line 34,  you use `Paths.get("/etc/path/to/my-resource.jar").toUri().toURL() ` for the sake of consistency I would do the same here :)